umm-spec-lib/src/cmr/umm_spec/acl_matchers.clj

```diff
@@ -22,20 +20,15 @@
   "The cache key for a URS cache."
   :collection-field-constraints)
 
-(defmulti matches-access-value-filter?
- "Returns true if the umm item matches the access-value filter"
- (fn [concept-type umm access-value-filter]
-   concept-type))
-
-(defmethod matches-access-value-filter? :collection
-  [concept-type umm access-value-filter]
+(defn matches-concept-access-value-filter?
```

Comment:
Is this the method that caused us trouble in runtime in SIT with the cache? That it couldn't find the method?

Reply:
I don't remember, but I rewrote it not to use multi functions, because it doesn't need it.  I was also able combine most of the code between the collection and the granule filter matching code.